lib/column/json.go

```diff
@@ -846,14 +847,18 @@ func (jCol *JSONObject) Rows() int {
 	return 0
 }
 
-// ClickHouse returns JSON as a tuple i.e. these will never be invoked
-
 func (jCol *JSONObject) Row(i int, ptr bool) interface{} {
-	panic("Not implemented")
+	if jCol.decoding != nil {
+		return jCol.decoding.Row(i, ptr)
+	}
+	panic("bug")
```

Comment:
what is exactly the case here? `Decode` func wasn't called prior?
let's make it more describtive